bouncer/templates/annotation.html.jinja2

```diff
@@ -1,17 +1,17 @@
 {% extends "templates/base.html.jinja2" %}
 
-{% set title = _("Hypothesis: Redirecting you&hellip;") %}
+{% set title = _("Hypothesis: Annotation for " ) + pretty_url  %}
```

Comment:
I think this actually needs to be something like:

```jinja2
{% set title = _("Hypothesis: Annotation for %(site)s" )|format(site=pretty_url)  %}
```

The reason is that in some languages the URL might not appear at the end of that sentence, so `%(site)s` needs to be part of the string that gets passed to `_()`, so that translators can move it around. (Unfortunately this also gives translators the chance to break things by removing or changing the `%(site)s`, but I think it's what we have to do.